pkg/plugins/manager/client/client.go

```diff
@@ -11,17 +11,20 @@ import (
 	"github.com/grafana/grafana/pkg/plugins/backendplugin"
 	"github.com/grafana/grafana/pkg/plugins/backendplugin/instrumentation"
 	"github.com/grafana/grafana/pkg/plugins/manager/registry"
+	"github.com/grafana/grafana/pkg/setting"
 )
 
 var _ plugins.Client = (*Service)(nil)
 
 type Service struct {
 	pluginRegistry registry.Service
+	cfg            *setting.Cfg
```

Comment:
If it's not too much trouble, would you mind using `*plugins.Cfg` as a dependency here? Feel free to add a new `bool` field for the feature toggle state. Then the `ProvideService` can turn the `*setting.Cfg` into the right thing with `plugins.FromGrafanaCfg`. It's our way of strangling the plugins package as much as possible 😄

Reply:
:+1: